### PR TITLE
Bump dependencies up to new major version, import API client from the SDK rather than the API. 

### PR DESCRIPTION
**Pull request by hello-ashleyintech**
### Type of change (place an x in the [ ] that applies)

- [ ] New sample
- [ ] New feature
- [ ] Bug fix
- [ ] Documentation

### Summary

Bump dependencies up to new major version, import API client from the SDK rather than the API. 

### Requirements (place an x in each [ ] that applies)

- [x] I’ve checked my submission against the Samples Checklist to ensure it complies with all standards
- [x] I have ensured the changes I am contributing align with existing patterns and have tested and linted my code
- [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct)

### CODE REVIEW

**Comment by seratch** [comment]
@hello-ashleyintech Is this an expected breaking change?
```
error: TS2339 [ERROR]: Property 'message_ts' does not exist on type 'WorkflowStepOutputs<{ message_link: { type: "string"; description: string; title: string; }; action: { type: "object"; description: string; title: string; }; interactivity: { type: CustomType<TypedObjectProperties, TypedObjectRequiredProperties<...>, { ...; }>; description: string; title: string; }; message_context:...'.
  messageTs: sendMessageStep.outputs.message_ts,
                                     ~~~~~~~~~~
    at file:///home/runner/work/deno-code-snippets/deno-code-snippets/Button_Interactions/workflows/interactive_blocks_demo.ts:52:3[8](https://github.com/slack-samples/deno-code-snippets/actions/runs/4623712800/jobs/8177811176?pr=7#step:7:9)
```

https://github.com/slack-samples/deno-code-snippets/actions/runs/4623712800/jobs/8177811176?pr=7

**Comment by seratch** [comment]
It's one of the outputs from the built-in SendMessage function: https://github.com/slack-samples/deno-code-snippets/blob/main/Button_Interactions/workflows/interactive_blocks_demo.ts#L52

**Comment by hello-ashleyintech** [comment]
@seratch Yep, I believe it's due to some breaking changes from the Deno dependencies being bumped to major versions. @filmaj will likely take a look at this sometime this week, along with the other samples that have failing CI checks due to the major version bump! We'll hold off on merging this for the CLI release tomorrow until it's resolved.

**Comment by seratch** [comment]
@hello-ashleyintech OK, the part needs to be `sendMessageStep.outputs.message_context.message_ts` instead. Can you resolve the error before merging this PR?

**Comment by seratch** [comment]
Ah, I got it. Thanks!

**Comment by seratch** [comment]
@hello-ashleyintech OK, one more error. It seems that we're dropping thread_ts support in the buil-in SendMessage function inputs. So, the code needs to be replaced with a custom function like my-send-message.ts
```
error: TS2769 [ERROR]: No overload matches this call.
  Overload 1 of 2, '(slackFunction: ISlackFunctionDefinition<{ channel_id: { type: "slack#/types/channel_id"; description: string; title: string; }; message: { type: "slack#/types/rich_text"; description: string; title: string; }; metadata: { type: "object"; ... 4 more ...; required: string[]; }; interactive_blocks: { ...; }; }, { ...; }, ("channel_id" | "message")[], ("message_context" | "message_link")[]>, inputs: WorkflowStepInputs<...>): TypedWorkflowStepDefinition<...>', gave the following error.
    Argument of type '{ channel_id: SingleParameterVariable; thread_ts: SingleParameterVariable; message: string; }' is not assignable to parameter of type 'WorkflowStepInputs<{ channel_id: { type: "slack#/types/channel_id"; description: string; title: string; }; message: { type: "slack#/types/rich_text"; description: string; title: string; }; metadata: { type: "object"; ... 4 more ...; required: string[]; }; interactive_blocks: { ...; }; }, ("channel_id" | "message")[]>'.
      Object literal may only specify known properties, and 'thread_ts' does not exist in type 'WorkflowStepInputs<{ channel_id: { type: "slack#/types/channel_id"; description: string; title: string; }; message: { type: "slack#/types/rich_text"; description: string; title: string; }; metadata: { type: "object"; ... 4 more ...; required: string[]; }; interactive_blocks: { ...; }; }, ("channel_id" | "message")[]>'.  Overload 2 of 2, '(functionReference: string, inputs: WorkflowStepInputs<ParameterSetDefinition, PossibleParameterKeys<ParameterSetDefinition>>): UntypedWorkflowStepDefinition', gave the following error.
    Argument of type 'SlackFunctionDefinition<{ channel_id: { type: "slack#/types/channel_id"; description: string; title: string; }; message: { type: "slack#/types/rich_text"; description: string; title: string; }; metadata: { type: "object"; ... 4 more ...; required: string[]; }; interactive_blocks: { ...; }; }, { ...; }, ("channel_id"...' is not assignable to parameter of type 'string'.
workflow.addStep(Schema.slack.functions.SendMessage, {
^
    at file:///home/runner/work/deno-code-snippets/deno-code-snippets/Event_Triggers/workflows/reply_to_reaction.ts:1[8](https://github.com/slack-samples/deno-code-snippets/actions/runs/4623960040/jobs/8178301104?pr=7#step:7:9):1
Error: Process completed with exit code 1.
```

**Comment by filmaj** [comment]
@seratch actually there is now a new built-in for this use case: `ReplyInThread`. All previous uses of `SendMessage` that used the `thread_ts` input should now instead be replaced with this new `ReplyInThread` built-in. It is kind of complicated, though, because `ReplyInThread` requires a `message_context` complex object type. Not all triggers provide `message_context` as a `{{data.*}}` property, unfortunately (though that is being worked on), so depending on the code, it may be needed to construct this `message_context` manually 😞

File: Event_Triggers/workflows/reply_to_reaction.ts
Comment (filmaj) [inline Event_Triggers/workflows/reply_to_reaction.ts:22]: This is what replacing `SendMessage` with `ReplyInThread` looks like as of SDK 2.0.

**Review by filmaj — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**
